meta-oe/recipes-connectivity/hiba/hiba_git.bb

```diff
@@ -0,0 +1,50 @@
+SUMMARY = "Google Host Identity Based Authorization"
+DESCRIPTION = "HIBA is a system built on top of regular OpenSSH \
+  certificate-based authentication that allows to manage flexible \
+  authorization of principals on pools of target hosts without the need to \
+  push customized authorized_users files periodically."
+
+S = "${WORKDIR}/git/hiba"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=7e161abe9a4585310725dd86c28d1ae2"
+
+PV = "git${SRCPV}"
+
+inherit autotools
+
+DEPENDS = "openssl openssh zlib"
+RDEPENDS:${PN} = " bash"
```

Comment:
We do not need the hiba-ca.sh on the target host, so 'bash' is not needed in RDEPENDS.